packages/config/config/devices/0x027a/zen32.json

```
@@ -276,6 +276,14 @@
 			"label": "Send Status Change Report: Timer",
 			"description": "Determine whether a trigger of this type should prompt a status change report to associated devices.",
 			"defaultValue": 1
+		},
+		{
+			"#": "27",
+			"$if": "firmwareVersion >= 2.40 && firmwareVersion < 10.0",
+			"$import": "~/templates/master_template.json#base_enable_disable_inverted",
+			"label": "LED Blinking When Buttons Are Pressed",
```

Let's call this:
```suggestion
			"label": "LED Indication: Button Pressed",
```

and the other one `"LED Indication: Settings Changed"`.

Also it might make sense to group those two together - the params don't need to be sorted.